### ISSUE TITLE
[iOS] 0.11.4 - ListView sections do not scroll to the absolute top of its container

### ISSUE THREAD

**Issue by jaygarcia** [facebook/react-native#3230]
I created a basic scaffold, copied "ListViewPagingExample.js" from master, pasted it into my 0.11.4 codebase as the only view, and noticed something very strange. The sections don't scroll as expected to the top of the view.

See below.

![listview](https://cloud.githubusercontent.com/assets/180031/10269571/e9ac137e-6aa9-11e5-99f4-850fb92f1551.gif)

Inside of an application without a header:
![listview](https://cloud.githubusercontent.com/assets/180031/10269583/4b1b71fe-6aaa-11e5-9c35-1434bf0e1e70.gif)

**Comment by jaygarcia** [comment]
What is very confusing is that this works perfectly within the confines of a Navigator component.   ListView is just one of those components that is so very mysterious sometimes :P

![listview](https://cloud.githubusercontent.com/assets/180031/10269625/1d0e385e-6aab-11e5-92b6-9cd2b9bb647e.gif)

**Comment by ide** [comment]
Turn off automatic adjustment of the content insets.

**Comment by jaygarcia** [comment]
@ide, thank you! I found similar issues after some more searches. 

This is clearly a documentation issue. I've tripped over this a few times already.  

automaticallyAdjustContentInsets is not a direct property of ListView. Without knowledge of the source, that is that ListView implements a ScrollView, how does a new person actually resolve this issue?

It seems that the ScrollView prop types (https://github.com/facebook/react-native/blob/master/Libraries/CustomComponents/ListView/ListView.js#L114) are included, but the documentation doesn't mention this at all.

I'd love to make a PR for this but to date, i've made several, all of which have been stalled or just ignored.

Thoughts?
